corehq/apps/builds/static/builds/js/edit-builds.js

```diff
@@ -1,7 +1,15 @@
-/* globals hqDefine */
-hqDefine('builds/js/edit-builds', function () {
-    var initial_page_data = hqImport('hqwebapp/js/initial_page_data').get,
-        doc = initial_page_data('doc'),
+hqDefine('builds/js/edit-builds', [
+    'jquery',
+    'knockout',
+    'underscore',
+    'hqwebapp/js/initial_page_data',
+], function (
+    $,
+    ko,
+    _,
+    initialPageData
+) {
+    var doc = initialPageData.get('doc'),
```

Comment:
nit: the indentation doesn't match with the next line

Reply:
eh? the next line is indented more because it's part of the same `var` statement